TwilioVerifySDK/TwilioVerify/Sources/Data/JWT/JwtGenerator.swift

```diff
@@ -20,7 +20,12 @@
 import Foundation
 
 protocol JwtGeneratorProtocol {
-  func generateJWT(forHeader header: [String: String], forPayload payload: [String: Any], withSignerTemplate signerTemplate: SignerTemplate) throws -> String
+  func generateJWT(
+    forHeader header: [String: String],
+    forPayload payload: [String: Any],
+    allowIphoneMigration: Bool,
```

Comment:
If we include `allowIphoneMigration` in the signer template, you could remove this new parameter